app/redux/reducers/misc.js

```diff
@@ -461,3 +480,33 @@ export const pendingReceivedInvites = (state = initialState.pendingReceivedInvit
       return state;
   }
 };
+
+export const dataDonationAccounts = (state = initialState.dataDonationAccounts, action) => {
```

Comment:
It looks like these `misc` reducers slipped through testing. (not that I can tell that there's anything amiss, but it's probably better to have them also unit tested)